.github/workflows/main.yml

```diff
@@ -5,8 +5,18 @@ on:
         branches: [ main ]
 
 jobs:
+  code-style:
+    name:    Code style check
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: PHP-CS-Fixer
+        uses: docker://oskarstark/php-cs-fixer-ga
+        with:
+          args: --rules=@PSR2 --dry-run -v .
```

Comment:
ahh...
Now I see :)
I didn't think about this solution, so no need for the `.php_cs.dist` file.
However I will leave it in the `rc-base` repo, for future reference.

Reply:
The tricky part for me was the iterator. In the [example](https://github.com/OskarStark/test-php-cs-fixer-ga/blob/master/.github/workflows/push.yml) action the default config was used, and that worked without the `.` iterator. But if you update the default config, like here, where the rule is specified, the iterator has to be also set.